Makefile.core.mk

```diff
@@ -140,12 +135,26 @@ update_all: update_ref_docs update_examples
 foo2:
 	hugo version
 
-.PHONY: init
-init:
+.PHONY: init preinit
+init: preinit
```

Comment:
just leaving a comment here for future reviewers digging in the history. Make is supposed to work LEFT->RIGHT, then TOP->BOTTOM. In prow, this is not the case. It appears prow is in some way running target operations out of roder. That is why this preinit step is here. This isn't a good long term solution but it will hold for now.

Reply:
@ericvn - did you follow the top to bottom rule of make? preinit should proceed init.

Reply:
Order fixed and added comment to Makefile as well.